public/resources/core/js/controllers/user.controller.js

```diff
@@ -59,11 +61,47 @@
         vm.isAdmin = isAdmin;
         vm.isSeller = isSeller;
         vm.isAuthenticated = isAuthenticated;
-	vm.cambiaCesvar = cambiaCesvar;
-	$scope.cesvar = 'Potorro';
+	vm.orgsVisible = orgsVisible;
+	vm.orgsInvisible = orgsInvisible;
+	vm.loggedAsIndividual = loggedAsIndividual;
+	vm.switchSession = switchSession;
+	vm.switchToUser = switchToUser;
+	vm.showOrgList = showOrgList;
+	vm.showOrgs = false;
+	vm.hasAdminRole = hasAdminRole;
+
+	function hasAdminRole() {
+	    var org = User.loggedUser.organizations.find(x => x.id == vm.currentUser.id);
+	    return !loggedAsIndividual() || org.roles.findIndex(x => x.name == "Admin") > -1;
+	}
+
+	function loggedAsIndividual() {
+	    return vm.currentUser.id != User.loggedUser.id;
+	}
+
+	function showOrgList(orgId) {
+	    return vm.currentUser.id != orgId;
+	}
+
+	function switchSession(orgId) {
+	    var currUser = User.loggedUser.organizations.find(x => x.id == orgId);
+	    vm.currentUser.name = currUser.name;
+	    vm.currentUser.email = currUser.email ? currUser.email : "email@example.com";
```

Comment:
I thought it was a required field for organizations and placed that default example email as a reminder.